### PR TITLE
[IMP] mail: allow seeing portal user profile from avatar popover

### PR DESCRIPTION
**Pull request by tsm-odoo**
In discuss, hovering over the avatar of the author's message opens a
popover, regrouping information about this user and allowing users to
view their profile. Until now, it was not possible to access portal
user's profile. This PR enables this feature.

task-3707734

### CODE REVIEW

**Comment by seb-odoo** [comment]
There are issue on runbot, but otherwise ok for the general idea. Waiting for runbot fix and testing to go further with the review.

**Comment by tsm-odoo** [comment]
Waiting for https://github.com/odoo/odoo/pull/159873 that will conflict

File: addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml [hunk 1]
```diff
@@ -32,6 +32,7 @@
                     <div class="justify-content-end d-flex align-items-baseline">
                         <div class="d-flex gap-2 o_avatar_card_buttons">
                             <button t-if="!user.share" class="btn btn-secondary btn-sm" t-on-click.stop="onSendClick">Send message</button>
+                            <button class="btn btn-secondary btn-sm" t-on-click.stop="onViewProfileClick">View Profile</button>
```
Comment (alexkuhn) [inline addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml:35]: `onViewProfileClick` => `onClickViewProfile`

**Review by alexkuhn — COMMENTED** [review]
LGTM otherwise

**Comment by seb-odoo** [comment]
Do we want to try to get that in the freeze? But new message jump is priority.

There are conflicts to resolve.

**Review by alexkuhn — APPROVED** [review]
LGTM otherwise
@robodoo delegate+

**Comment by tsm-odoo** [comment]
@robodoo r+

**Comment by tsm-odoo** [comment]
@robodoo r+

:point_up: rebased

**Pull request closed without merging**